src/main/java/org/cactoos/http/HtContentType.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package org.cactoos.http;
+
+import org.cactoos.Input;
+import org.cactoos.Scalar;
+
+/**
+ * Content-Type of HTTP response.
+ *
+ * @author Victor Noel (victor.noel@crazydwarves.org)
+ * @version $Id$
+ * @since 0.1
+ */
+public final class HtContentType implements Scalar<String> {
+
+    /**
+     * Response head part.
+     */
+    private final Input head;
+
+    /**
+     * Ctor.
+     * @param head Response head part
+     */
+    public HtContentType(final Input head) {
+        this.head = head;
+    }
+
+    @Override
+    public String value() {
+        return new HtHeaders(this.head).get("content-type");
```

Comment:
Ok. Actually I made some research on the matter, and according to http://mattryall.net/blog/2008/03/default-content-type, it would be better to even return `application/octet-stream`. WDYT?